lib/api/bucketGetNotification.js

```diff
@@ -0,0 +1,65 @@
+const { metadataValidateBucket } = require('../metadata/metadataUtils');
+const { pushMetric } = require('../utapi/utilities');
+const collectCorsHeaders = require('../utilities/collectCorsHeaders');
+const { NotificationConfiguration } = require('arsenal').models;
+
+/**
+ * Format of xml response:
+ *
+ * <ONotificationConfiguration>
```

Comment:
Extra O at the start
```suggestion
 * <NotificationConfiguration>
```